rllib/utils/exploration/stochastic_sampling.py

```diff
@@ -91,8 +90,10 @@ def _get_tf_exploration_action_op(self, action_dist, timestep, explore):
             false_fn=lambda: deterministic_actions)
 
         def logp_false_fn():
```

Comment:
should we make logp_false_fn a util fn in this file, so we can call it 3 times, and also give it some descriptions to explain things a little.

Reply:
Great idea, thanks @gjoliver. Will do!

Reply:
Done, created a new util func in `utils/tf_ops.py`. Waiting for tests. Also moved the Random exploration component's logic into a new util func, which even takes care now of complex actions (arbitrarily nested dict/tuples of tensors).